app/controllers/api/messaging.php

```diff
@@ -143,6 +144,7 @@
     ->label('audits.resource', 'provider/{response.$id}')
     ->label('event', 'providers.[providerId].create')
     ->label('scope', 'providers.write')
+    ->label('resourceType', 'messaging')
```

Comment:
It would be useful to have more granularity for messaging. I think the resource types here need to be `providers` , `topics`, `subscribers` and `messages` ?